atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraClientPoolingContainer.java

```diff
@@ -270,5 +270,43 @@ private void registerMetrics(GenericObjectPool<Client> pool, String metricPrefix
                 CassandraClientPoolingContainer.class,
                 metricPrefix, "proportionDestroyedByBorrower",
                 () -> ((double) pool.getDestroyedByBorrowValidationCount()) / ((double) pool.getCreatedCount()));
+
+        // Count of active or idle threads in the pool
+        metricsManager.registerMetric(
+                CassandraClientPoolingContainer.class,
+                metricPrefix, "numActive",
+                pool::getNumActive);
+        metricsManager.registerMetric(
+                CassandraClientPoolingContainer.class,
+                metricPrefix, "numIdle",
+                pool::getNumIdle);
+
+        // Proportion of active and idle threads in the pool
+        metricsManager.registerMetric(
+                CassandraClientPoolingContainer.class,
+                metricPrefix, "proportionActiveThreads",
+                () -> ((double) pool.getNumActive()) / ((double) pool.getCreatedCount()));
```

Comment:
nit: because of widening conversions casting one should be fine. But anyway let's get this in, we can worry about cleaning it up later.